docs/src/Menu.tsx

```diff
@@ -1,6 +1,19 @@
 import { useState } from 'react';
 import type { FunctionsData } from '../scripts/transform';
 
+const BADGE_COLORS: Record<string, string> = {
+  Array: 'bg-emerald-500',
+  Object: 'bg-amber-500',
+  Function: 'bg-fuchsia-500',
+  Number: 'bg-blue-500',
+  String: 'bg-violet-500',
+  Guard: 'bg-rose-500',
+  Type: 'bg-stone-500',
+  Other: 'bg-gray-500',
+};
```

Comment:
You don't need the type definition, simply set it to satisfy the API you need
```suggestion
const BADGE_COLORS = {
  Array: 'bg-emerald-500',
  Object: 'bg-amber-500',
  Function: 'bg-fuchsia-500',
  Number: 'bg-blue-500',
  String: 'bg-violet-500',
  Guard: 'bg-rose-500',
  Type: 'bg-stone-500',
  Other: 'bg-gray-500',
} satisfies Record<string, `bg-${string}-${number}`>;
```

Reply:
Thanks for the suggestion @eranhirsch!

Unfortunately, this gives me an error when accessing the index (hence the solution with explicit type `Record<string, string>`):
```ts
BADGE_COLORS[item?.category ?? '']
// expression of type 'string' can't be used to index type '{ Array: "bg-emerald-500"; Object: "bg-amber-500"; Function: "bg-fuchsia-500"; Number: "bg-blue-500"; String: "bg-violet-500"; Guard: "bg-rose-500"; Type: "bg-stone-500"; Other: "bg-gray-500"; }'.
```

I guess it might be solved by getting proper typing for `item.category` (union type `'Array' | 'Object' | 'Number'` etc.) - any ideas? :slightly_smiling_face: